Description: Upstream changes introduced in version 12.0.0-1~ppa1~maverick1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 twisted-lore (12.0.0-1~ppa1~maverick1) maverick; urgency=low
 .
   * New upstream version.
   * Rebuild package for use in PPA.
 .
 The person named in the Author field signed this changelog entry.
Author: Jessica McKellar <jesstess@mit.edu>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- twisted-lore-12.0.0.orig/doc/man/lore.1
+++ twisted-lore-12.0.0/doc/man/lore.1
@@ -55,8 +55,8 @@ Display version information and exit.
 .TP
 \fI--config\fR
 Add input/output-specific information. 
-HTML output allows for 'ext=<extension>', 'template=<template>'
-and 'baseurl=<format string for API URLs>'. LaTeX
+HTML output allows for 'ext=<extension>',
+'template=<template>' and 'baseurl=<format string for API URLs>'. LaTeX
 output allows for 'section' or 'chapter' in Lore, and nothing in Math-Lore. 
 Lore output allows for 'ext=<extension>'. Lint output allows nothing.
 Note that disallowed \fI--config\fR options are merely ignored, and do
